app/models/manageiq/providers/container_manager/container_template.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::ContainerManager::ContainerTemplate < ContainerTemplate
```

Comment:
[yay we resolved it, given https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/81 now https://github.com/ManageIQ/manageiq-providers-openshift/pull/28 passes.]